core/src/test/java/org/elasticsearch/action/support/replication/ClusterStateCreationUtils.java

```diff
@@ -145,6 +145,44 @@ public static ClusterState state(String index, boolean activePrimaryLocal, Shard
     }
 
     /**
+     * Creates cluster state with and index that has #(shardStates) started primary shards and no replicas, using the given number of nodes.
```

Comment:
I don't understand this, what do you mean by #(shardStates) started primary shards?